wallet/src/de/schildbach/wallet/ui/LockScreenActivity.kt

```diff
@@ -530,15 +531,17 @@ open class LockScreenActivity : SecureActivity() {
         }
     }
 
-    open fun onLockScreenActivated() {
+    private fun handleLockScreenActivated() {
         if (this::alertDialog.isInitialized){
             alertDialog.dismissDialog()
         }
-
         lockScreenBroadcaster.activatingLockScreen.call()
         dismissDialogFragments(supportFragmentManager)
+        onLockScreenActivated()
     }
 
+    open fun onLockScreenActivated() { }
```

Comment:
This will fix the issue with dialogs not being hidden if `onLockScreenActivated` is overridden and `super` isn't called.